### PR TITLE
Run tests in all Rust channels

### PR DESCRIPTION
**Pull request by upsuper**

### CODE REVIEW

**Comment by bryant** [comment]
Rather than adding a separate feature flag, I'm considering placing the `constant_eq` tests (they're not really benchmarks, per se) in their own module under `$ROOT/benches` so that they're still accessible via `cargo bench` with the right rustc: https://github.com/bryant/argon2rs/pull/7 . Let me know your thoughts on that.

However, I would still like to merge the changes you've made to `.travis.yml`, so would you mind rebasing your changes off #7?

**Comment by upsuper** [comment]
Could you merge that first?

Also conditionally running benchmarks could be a bit challenging. I'm not quite familiar with travis. Probably your current method is fine enough, and the only thing left is adding beta to the list.

I'm not quite sure whether running benchmark test on travis is a good idea, though.

**Comment by bryant** [comment]
Done.

File: benches/constant_eq.rs [hunk 1]
```diff
@@ -1,5 +1,6 @@
 // demonstrates (to some degree of certainty modulo process scheduling) that the
 // run time of `verifier::constant_eq` is independent of its inputs.
+#![cfg(nightly)]
```
Comment (upsuper) [inline benches/constant_eq.rs]: This actually doesn't work at all... It just make the whole mod a no-op. I'm guessing it is currently not possible to detect nightly in code.

Comment (bryant) [inline benches/constant_eq.rs]: Yes. This is why detection is instead done in `travis.sh` via `TRAVIS_RUST_VERSION`.

**Comment by bryant** [comment]
I think this was fixed by https://github.com/bryant/argon2rs/commit/cfe2fac1bb0a8f896bac0732a0bbb0aac020925c and https://github.com/bryant/argon2rs/commit/2ed4cadce2bca8e7e785d4333d6d3d2004871367 . Feel free to re-open this if you spot any problems.

**Pull request closed without merging**